runtime/src/main/resources/application.properties

```diff
@@ -71,6 +71,7 @@ huaweicloud.european.price.calculator.url=https://portal.eu.huaweicloud.com/api/
 huaweicloud.chinese.price.calculator.url=https://portal.huaweicloud.com/api/cbc/global/rest/BSS/billing/ratingservice/v2/inquiry/resource
 enable.agent.api.only=false
 agent.api.end.point=http://localhost:8080
+support.default.deployment.tool.versions.only=true
 deployer.terraform.install.dir=/opt/terraform
 deployer.terraform.versions=1.6.0,1.7.0,1.8.0,1.9.0
```

Comment:
can we rename this to deployer.terraform.default.supported.versions pls.
and for opentofu - deployer.opentofu.default.supported.versions pls

Reply:
renamed